src/hotspot/share/oops/flatArrayKlass.hpp

```diff
@@ -92,7 +92,7 @@ class FlatArrayKlass : public ObjArrayKlass {
   static jint array_layout_helper(InlineKlass* vklass, LayoutKind lk); // layout helper for values
 
   // sizing
-  static int header_size()  { return sizeof(FlatArrayKlass)/HeapWordSize; }
+  static int header_size()  { return sizeof(FlatArrayKlass) / wordSize; }
```

Comment:
Curiously enough we actually got this right for `RefArrayKlass::header_size()` and I'm guessing the following assertion didn't actually do anything meaningful:
```cpp
assert(RefArrayKlass::header_size() <= InstanceKlass::header_size(),
         "array klasses must be same size as InstanceKlass");
```